primeqa/mrc/run_mrc.py

```diff
@@ -128,6 +129,16 @@ class DataTrainingArguments:
             "help": "The configuration name of the dataset to use (via the datasets library)."
         }
     )
+    dataset_filter_column_name: str = field(
```

Comment:
This argument needs more documentation and it is not easy to understand what it does. Is it the right place for this argument? Can it be handled by pre-processors ?